### DISCUSSION TITLE
Docker-compose quick start doesn't work with logging_level: DEBUG

### DISCUSSION THREAD

**Discussion by GApfel** [apache/airflow#23149] [General] [answered]
### What do you see as an issue?

I'm using the quickstart docker-compose.yml (https://github.com/apache/airflow/tree/main/docs/apache-airflow/start/docker-compose.yaml), and I wanted to change the log_level to DEBUG by adding `AIRFLOW__LOGGING__LOGGING_LEVEL: 'DEBUG'` to the `x-airflow-common` environment. This breaks the `airflow-init` container because the line `airflow_version=$$(gosu airflow airflow version)` no longer contains only the airflow version, but a lot of DEBUG information as well:

`[2022-04-20 07:48:03,127] {settings.py:219} DEBUG - Setting up DB connection pool (PID 8) [2022-04-20 07:48:03,127] {settings.py:314} DEBUG - settings.prepare_engine_args(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=8 [2022-04-20 07:48:03,248] {cli_action_loggers.py:40} DEBUG - Adding <function default_action_log at 0x7f2a9a991710> to pre execution callback 2.2.3 [2022-04-20 07:48:03,293] {settings.py:340} DEBUG - Disposing DB connection pool (PID 8)`

The next line attempts to format this as a number, but fails:
`airflow_version_comparable=$$(ver $${airflow_version})
environment: line 1: printf: [2022-04-20: invalid number`

The result is that the version comparison fails, and the container exits with the message:
`ERROR!!!: Too old Airflow version [2022-04-20 07:48:03,127] {settings.py:219} DEBUG - Setting up DB connection pool (PID 8)
[2022-04-20 07:48:03,127] {settings.py:314} DEBUG - settings.prepare_engine_args(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=8
[2022-04-20 07:48:03,248] {cli_action_loggers.py:40} DEBUG - Adding <function default_action_log at 0x7f2a9a991710> to pre execution callback
2.2.3
[2022-04-20 07:48:03,293] {settings.py:340} DEBUG - Disposing DB connection pool (PID 8)!
The minimum Airflow version supported: 2.2.0. Only use this or high`

Notice that all the DEBUG information from `airflow_version` is printed here, instead of just the version.

### Solving the problem

I think there should be some option to suppress logging from `gosu` even while using DEBUG level logging for everything else. Either that or some regex or parsing should be used to extract the airflow version from the string generated by the gosu command.

### Anything else

This happens whenever I use `AIRFLOW__LOGGING__LOGGING_LEVEL: 'DEBUG'`, but I've tried other log levels (`'INFO'`, `'WARNING'`) and they work without a problem.

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment, marked as answer]
We are not likely to fix it (it's not a bug) - Quick-Start is quick start and you cannot expect it to work after you modify it (even to add debugging this way). 

But if you find an easy way to change  it - feel free to provide a PR. For example you coudd add 

`AIRFLOW__LOGGING__LOGGING_LEVEL=INFO` in front of `airlfow version` command.

I think it might work.

**Reply by GApfel** [reply]
Thank you, I did get this to work. I have submitted a PR #23360
